### DISCUSSION TITLE
使用一个大网盘和多个小网盘进行负载均衡时无法解析文件链接的问题

### DISCUSSION THREAD

**Discussion by colorfulkurumi** [AlistGo/alist#4514] [Q&A] [answered]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

v3.18.0

### Driver used / 使用的存储驱动

OneDrive & SharePoint

### Describe the bug / 问题描述

我试图使用一个无限存储的SharePoint全局和一个E5的若干个子号进行负载均衡，SP里每个文件夹对应一个不同的E5子号。
![image](https://github.com/alist-org/alist/assets/133512333/62b3df06-fce5-4c51-a123-bd2f8782ce87)
但经过测试，如图所示的网盘配置会导致所有的文件都无法被正常解析
![image](https://github.com/alist-org/alist/assets/133512333/69344ace-fefa-4158-ad0d-b807f295195a)
我只设置一个E5与SP负载均衡的时候，E5里存在的文件可以正常负载均衡，不存在的就无法解析，如果把第二个E5的负载均衡编号设置为balance2的话依然所有文件都无法正常解析。
请问这是一个bug吗，还是说像我这样的用途需要其他的设置方式？

### Reproduction / 复现链接

https://yhsxsx.top/helpme/
专为提交此issue打开的所有子号使用相同的balance编号的子目录，表现为文件正常显示但全部无法解析

### Logs / 日志

在所有报错产生的时间范围内cli没有任何报错信息

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xhofe** [comment, marked as answer]
负载均衡的`Mount Path`仅支持在最后一级添加`.balance`。

**Reply by colorfulkurumi** [reply]
好吧，感谢回复
